catalog-rest-service/src/main/java/org/openmetadata/catalog/elasticsearch/ElasticSearchIndexDefinition.java

```diff
@@ -197,6 +200,8 @@ public ElasticSearchIndexType getIndexMappingByEntityType(String type) {
       return ElasticSearchIndexType.USER_SEARCH_INDEX;
     } else if (type.equalsIgnoreCase(Entity.TEAM)) {
       return ElasticSearchIndexType.TEAM_SEARCH_INDEX;
+    } else if (type.equalsIgnoreCase(Entity.TOPIC)) {
```

Comment:
Isn't this code unreachable?

There is an Entity.TOPIC above where it returns TOPIC_SEARCH_INDEX

Reply:
That is a merge error. It should be Glossary